docs/conf.py

```diff
@@ -137,7 +137,7 @@ def word_filter(self, word):
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'bootstrap'
+html_theme = 'clusterhq'
```

Comment:
Because version control.  That is, if we ever want the old theme, we can recover it from version control and having it around doing nothing clutters things up, and suggests that it might have some purpose in the current documentation when it doesn't.